django_pivot/tests/pivot/test.py

```diff
@@ -189,6 +187,43 @@ def test_pivot_aggregate(self):
                 avg = sum(spends) / len(spends) if spends else 0
                 self.assertAlmostEqual(row[dt], float(avg), places=4)
 
+    def test_pivot_display_transform(self):
+        def display_transform(string):
+            return 'prefix_' + string
+        shirt_sales = ShirtSales.objects.all()
+
+        pt = pivot(ShirtSales.objects.all(), 'style', 'gender', 'units', display_transform=display_transform)
+
+        for row in pt:
+            style = row['style']
+            for gender in genders:
+                gender_display = display_transform('Boy' if gender == 'B' else 'Girl')
+                self.assertEqual(row[gender_display], sum(ss.units for ss in shirt_sales if ss.style == style and ss.gender == gender))
+
+    def test_pivot_multiple_rows(self):
+        shirt_sales = ShirtSales.objects.all()
+
+        pt = pivot(ShirtSales.objects.all(), ('style', 'store'), 'gender', 'units')
+
+        for row in pt:
+            style = row['style']
+            store = row['store']
+            for gender in genders:
+                gender_display = 'Boy' if gender == 'B' else 'Girl'
+                self.assertEqual(row[gender_display], sum(ss.units for ss in shirt_sales if ss.style == style and ss.gender == gender and ss.store_id == store))
+
+    def test_pivot_on_choice_field_row_with_multiple_rows(self):
+        shirt_sales = ShirtSales.objects.all()
+
+        pt = pivot(ShirtSales.objects.all(), ('gender', 'store'), 'style', 'units')
```

Comment:
This works when `'gender'` is the first column, we get results like
```
{'gender': 'B', 'store': 2, 'get_gender_display': 'Boy', 'Fancy': 81, 'Golf': 84, 'Tee': 83}
```
So the user can get the display value `'Boy'`. But if you call `pivot` with your rows in the other order:
```
pt = pivot(ShirtSales.objects.all(), ('store', 'gender'), 'style', 'units')
```
We get rows like this
```
{'store': 1, 'gender': 'G', 'Fancy': 78, 'Golf': 80, 'Tee': 84}
```
With no `get_gender_display` entry.

I think the issue is in pivot.py. We don't need line 23 (`row = values[0]`) we need to put a loop around lines 31 through 36:
```
for row in values:
   # lines 31 through 36 here.
```